src/main/java/com/learnreactivespring/handler/user/UserHandler.java

```diff
@@ -0,0 +1,70 @@
+package com.learnreactivespring.handler.user;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.web.reactive.function.BodyInserters.fromValue;
+
+import com.learnreactivespring.model.City;
+import com.learnreactivespring.model.User;
+import com.learnreactivespring.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.server.ServerRequest;
+import org.springframework.web.reactive.function.server.ServerResponse;
+import reactor.core.publisher.Mono;
+
+@Component
+public class UserHandler implements IUserHandler {
+
+  @Autowired
+  UserRepository userRepository;
```

Comment:
this should be private and please use constructor injection where ever applicable.